app/models/favorite.rb

```diff
@@ -13,7 +13,26 @@ def add_pet(id)
     @contents[id.to_s] = count_of(id) + 1
   end
 
+  #if it's already favorited, dont add
+  #if its not already favorited, add pet
+  def already_favorited?(id)
```

Comment:
note for next round:
this might be a method on its own but when i tried it, it said a hash doesnt have access to the method but it worked in pry?